examples/chatpromptbuilder.py

```diff
@@ -0,0 +1,37 @@
+from haystack.components.builders import ChatPromptBuilder
+from haystack.components.generators.chat.openai import OpenAIChatGenerator
+from haystack.dataclasses.chat_message import ImageContent
+
+### Use multimodal inputs with ChatPromptBuilder
+
+template = """
+{% message role="system" %}
```

Comment:
If we do end up doing something like this how would we also pass things like meta and name?

Reply:
We could potentially support something like this:
`{% message role="system"  name="a name" meta={"key": "value"}%}`

`name` should be configurable.
Currently, I don't have specific use cases in mind for configuring `meta` inside a template but should be feasible.